README.md

```diff
@@ -105,6 +105,88 @@ here.
   - Only unary calls are supported for now.
 
 
+### Bazel
```

Comment:
1. Can we move this section between "TypeScript Support" and "Proxy Interoperability"
 2. Can we use two hashes headings "## Bazel Support"?